src/jarque_bera.jl

```diff
@@ -63,7 +63,7 @@ function JarqueBeraTest(y::AbstractVector{T}) where T<:Real
         m4r += yi^4 / n
     end
     # compute central moments (http://mathworld.wolfram.com/CentralMoment.html)
-    m2 = -m1r^2 + m2r
+    m2 = abs(-m1r^2 + m2r)
```

Comment:
I wonder what the (dis)advantages of this approach are compared with

```suggestion
    m2 = max(-m1r^2 + m2r, 0)
```

Reply:
I was just about to pull a request for that option! It is better.